mwpc/mwpc1/R3BMwpc1Cal2Hit.h

```diff
@@ -28,6 +28,8 @@
 
 #define Mw1PadsX 64
 #define Mw1PadsY 40
+#define S522 522
+#define S509 509
```

Comment:
Using C macro is a bad idea. If you just want to define some int variables, please do like this:
```cpp
constexpr int Mw1PadsX = 64;
```
And place them in the source file (.cxx). But of course if you really need them in the header file, you could also place them over there.

Reply:
There is no reason for this to be in the header file, has been moved to the source file.